app/models/submission.rb

```diff
@@ -91,6 +91,8 @@ def saveFile(upload)
       # Sanity!
       upload['file'].rewind
       File.open(path,"wb") { |f| f.write(upload['file'].read)}
+    elsif upload['local_submit_file']
+      File.open(path,"wb") { |f| f.write(IO.read(upload['local_submit_file']))}
```

Comment:
upload['file'] is a class that we're not using in the case of local submit. so there is not way to do `.read`